tests/test.py

```diff
@@ -1,30 +1,72 @@
-from asaps.models import Client
```

Comment:
In order for pytest to automatically discover this test module it needs to be named `test_<something>.py`. By convention you usually create a separate test module for every module in your package, so this one would be called `test_models.py`.